core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -21,31 +20,31 @@ import akka.persistence.cassandra.query.scaladsl.CassandraReadJournal
 import akka.persistence.cassandra.session.scaladsl.CassandraSession
 import akka.persistence.journal.{ AsyncWriteJournal, Tagged }
 import akka.persistence.query.PersistenceQuery
-import akka.serialization.{ Serialization, SerializationExtension }
+import akka.serialization.{ AsyncSerializer, Serialization, SerializationExtension }
 import akka.stream.ActorMaterializer
+import akka.util.OptionVal
 import com.datastax.driver.core._
 import com.datastax.driver.core.exceptions.DriverException
 import com.datastax.driver.core.policies.RetryPolicy.RetryDecision
 import com.datastax.driver.core.policies.{ LoggingRetryPolicy, RetryPolicy }
 import com.datastax.driver.core.utils.{ Bytes, UUIDs }
 import com.typesafe.config.Config
-import scala.collection.immutable.Seq
+
 import scala.collection.JavaConverters._
+import scala.collection.immutable
+import scala.collection.immutable.Seq
 import scala.concurrent._
 import scala.util.control.NonFatal
 import scala.util.{ Failure, Success, Try }
 
-import akka.serialization.AsyncSerializer
-import akka.util.OptionVal
-
 class CassandraJournal(cfg: Config) extends AsyncWriteJournal
```

Comment:
Could you add an `@DoNotInherit` on this and put a comment explaining that we don't support extensions and don't guarantee source compatibility if you do?

Reply:
ok